src/App.tsx

```diff
@@ -1,27 +1,93 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+
+import Paper from '@mui/material/Paper';
+import Button from '@mui/material/Button';
+
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
+
+import { getAll, get5First, getRedGoods } from './api/goods';
+
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+
+  const handleListOfGoods = () => {
+    getAll()
+      .then((good) => {
+        setGoods(good);
+      });
+  };
```

Comment:
you can make it shorter 
const handleListOfGoods = () => {
  getAll()
    .then(setGoods);
};

Reply:
i've' done only for better understanding and education as Vova said